backend/data/categories.yaml

```diff
@@ -10,6 +10,10 @@ categories:
   - id: starter
     name: Starter
 
+spongecategory:
+  - id: sponge
+    name: Sponges
```

Comment:
Why does this work entirely different from the other categories ? Please add an `alt` category in the `categories` section above and call it `Alts`

Reply:
Because I needed to seperate 2 usescases, for invites only & for the waitlist page categories

Reply:
Ah I see, lets just rename it to `altcategory` then and I'm happy